### PR TITLE
Update zh.js & zhtr.js (Quality bump, v5.4.2)

### PR DESCRIPTION
**Pull request by Kxlici**

### CODE REVIEW

**Review by 0v0katai — CHANGES_REQUESTED** [review]

File: zh.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ settings.local.region 默认地区
 settings.local.lang 语言
 settings.network.header 网络
 settings.network.netRerouting 动态路由（影响网络延时）
-settings.network.netRate 网络刷新率 (Hz)
+settings.network.netRate 网络刷新率 (赫兹）
```
[the remark this answers is not in the document]
Comment (mty22) [reply zh.js:94]: > @mty22 Should we leave "Hz" in English or can it be translated as 赫兹? (They have the exact same meaning)

@0v0katai / @Kxlici - Please leave 'Hz' in English.

**Comment by 0v0katai** [comment]
Also, please change line 40 to ”个粉丝“ too (#401)

File: zh.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ generic.delete 删除
 generic.save 保存
 generic.load 加载
 generic.search 搜索
-generic.leave 离开
+generic.leave 退出
```
[the remark this answers is not in the document]
Comment (Kxlici) [reply zh.js:23]: > The exact meaning of 退出 is quit, while 离开 means "leave".

they actually both means leave and i think that 退出 is more accurate

File: zh.js [hunk 3]
```diff
@@ -42,7 +42,7 @@ generic.following 个关注中
 generic.games 游玩场数
 generic.wins 胜场数
 generic.lose 败场数
-generic.clan 公会
+generic.clan 兵团
```
[the remark this answers is not in the document]
Comment (Kxlici) [reply zh.js:45]: 公会 more means for club and not clan, i think 兵团 is More accurate

**Comment by Kxlici** [comment]
please review again, i did some fixes

File: zh.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (chewyjm) [reply zh.js:45]: 公会 will be better translation for clan

**Review by 0v0katai — CHANGES_REQUESTED** [review]

File: zh.js [hunk 4]
```diff
@@ -941,7 +941,7 @@ modes.prophunt.rotation.off ［物品猎杀］已禁用物品旋转
 modes.prophunt.rotation.reset ［物品猎杀］已重置物品旋转。
 commands.login 登录以使用此命令
 commands.connected {0} 个玩家已连接，{1} 个玩家在标题画面
-commands.host {0} 是当前服主。
+commands.host {0} 是当前房主。
```
Comment (0v0katai) [inline zh.js:944]: Since we've used "服务器" in the previous lines "服主" is more appropriate.

**Review by 0v0katai — CHANGES_REQUESTED** [review]

File: zh.js (hunk 4 above)
Comment (Kxlici) [reply zh.js:944]: then we can change "服务器" to ‘’房间‘’

Comment (Kxlici) [reply zh.js:944]: they basically mean the same thing and "服务器" sounds abit too formal

File: zh.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (0v0katai) [reply zh.js]: image
image

File: zh.js (hunk 4 above)
Comment (chewyjm) [reply zh.js:944]: 服主 will be better as we used 'server' in English version.

**Comment by mty22** [comment]
@0v0katai can you please ping me when ready to merge. :) thanks

File: zh.js (hunk 4 above)
Comment (mty22) [reply zh.js:944]: Let's stick with `服主`

File: zh.js [hunk 5]
```diff
@@ -844,82 +844,82 @@ stats.thrownkills 投掷击杀数
...
-windows.playerlist.punish 你确定要对 {0} 采取措施吗？
+twitch.link 你确定要链接你的Twitch账户吗？
+twitch.unlink 你确定要取消链接你的Twitch账户吗？
+windows.clan.leave 你确定要离开此战队吗？
+windows.clan.delete 你确定要解散此战队吗？
+windows.clan.kick 你确定要从此战队踢出{0}吗？
+windows.store.premium.purchase 你确定要购买{0}天的会员吗？
+windows.store.premium.gift 你确定要赠送'{0}'{1}天的会员吗？
+windows.playerlist.ban 你确定要从此服务器封禁并踢出{0}吗？
+windows.playerlist.flag 你确定要封禁{0}吗？
```
Comment (0v0katai) [inline zh.js:864]: Both 标记 and 封禁 are not the clear words to describe 'hackertag'. Now I have 2 proposals on this line:
- 你确定要标记 {0} 为外挂玩家吗？(Are you sure you want to flag {0} as a hacker?)
- 你确定要为 {0} 贴上外挂玩家标签吗？(Are you sure you want to label {0} with hacker tag?)
The first one has some consistency with en.js as it used 'flag', while the second one is a sense-for-sense translation  (as flag = hackertag)

**Review by 0v0katai — CHANGES_REQUESTED** [review]

File: zh.js (hunk 5 above)
Comment (0v0katai) [reply zh.js:864]: Let's go for the first proposal.

**Review by 0v0katai — APPROVED** [review]
@mty22 Ready to merge ;)

**Comment by mty22** [comment]
@Kxlici can you please reach out to me with your IGN? (ping me here or discord dm)

**Pull request merged**